src/zenml/stack/stack.py

```diff
@@ -358,10 +358,19 @@ def deploy_pipeline(
         )
         start_time = time.time()
 
+        original_cache_boolean = pipeline.enable_cache
+        if 'enable_cache' in runtime_configuration:
+            pipeline.enable_cache = runtime_configuration.pop('enable_cache')
```

Comment:
Maybe add a log here explaining that the RunTimeConfiguration is used but that the global enable_cache is retained